functions/Copy-DbaCredential.ps1

```diff
@@ -1,57 +1,52 @@
 ﻿function Copy-DbaCredential {
 	<#
 		.SYNOPSIS
-			Copy-DbaCredential migrates SQL Server Credentials from one SQL Server to another, while maintaining Credential passwords.
+			Copy-DbaCredential migrates SQL Server Credentials from one SQL Server to another while maintaining Credential passwords.
 
 		.DESCRIPTION
-			By using password decryption techniques provided by Antti Rantasaari (NetSPI, 2014), this script migrates SQL Server Credentials from one server to another, while maintaining username and password.
-
-			Credit: https://blog.netspi.com/decrypting-mssql-database-link-server-passwords/
-			License: BSD 3-Clause http://opensource.org/licenses/BSD-3-Clause
+			By using password decryption techniques provided by Antti Rantasaari (NetSPI, 2014), this script migrates SQL Server Credentials from one server to another while maintaining username and password.
 
 		.PARAMETER Source
-			Source SQL Server (2005 and above). You must have sysadmin access to both SQL Server and Windows.
+			Source SQL Server. You must have sysadmin access and server version must be SQL Server version 2000 or newer.
 
 		.PARAMETER SourceSqlCredential
-			Allows you to login to servers using SQL Logins as opposed to Windows Auth/Integrated/Trusted. To use:
+			Allows you to login to servers using SQL Logins instead of Windows Authentication (AKA Integrated or Trusted). To use:
 
 			$scred = Get-Credential, then pass $scred object to the -SourceSqlCredential parameter.
 
-			Windows Authentication will be used if DestinationSqlCredential is not specified. SQL Server does not accept Windows credentials being passed as credentials.
+			Windows Authentication will be used if SourceSqlCredential is not specified. SQL Server does not accept Windows credentials being passed as credentials.
+
 			To connect as a different Windows user, run PowerShell as that user.
-	
-		.PARAMETER Credential
```

Comment:
This breaks the command by removing it. It is part of the parameter list so the test are failing as it was removed from help.